dbt_semantic_interfaces/validations/metrics.py

```diff
@@ -120,8 +120,10 @@ def validate_manifest(cls, semantic_manifest: SemanticManifestT) -> Sequence[Val
                 )
 
             if window:
-                issues += cls.validate_metric_time_window(
-                    metric_context=metric_context, window=window, custom_granularities=custom_granularity_names
+                issues.extend(
```

Comment:
I don't know why the typechecker would accept one of these and not the other, but tbh, I prefer extend here anyway so I'm not going to complain :P

Reply:
you can't do `list += sequence` but you CAN pass a sequence into `extend()`!